.github/workflows/release.yml

```diff
@@ -18,34 +18,122 @@ jobs:
       tag: ${{ steps.get-tag.outputs.tag }}
       last: ${{ steps.get-last-tag.outputs.last }}
       notes: ${{ steps.get-release-notes.outputs.notes }}
+      result: ${{ steps.py-get-updates.outputs.result }}
+      mails: ${{ steps.get-user-emails.outputs.mails }}
+      authors: ${{ steps.get-authors.outputs.authors }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           fetch-depth: '0'
+          
+      - name: Install python
+        uses: actions/setup-python@v2
         
       - name: Get release tag
         id: get-tag
         run: |
           MERGES=$(gh pr list -B main -s merged --search "closed:>2021-02-28" -L 9999 | wc -l | awk '{print $1}')
           echo "::set-output name=tag::release-$MERGES"
-          
+                
       - name: Get last release
         id: get-last-tag
         run: |
           LAST=$(gh release list -L 1 | awk '{print $1}')
           echo "::set-output name=last::$LAST"
           
+      - name: Get User emails
+        id: get-user-emails
+        run: |
+          echo MAILS=$(git log ${{ steps.get-last-tag.outputs.last }}..HEAD --pretty="%ae" --merges) >> $GITHUB_ENV
+          
+      - name: Get Authors
+        id: get-authors
+        run: |
+          AUTHORS=$(git log ${{ steps.get-last-tag.outputs.last }}..HEAD --pretty="%an" --merges)
+          AUTHORS="${AUTHORS//'%'/'%25'}"
+          AUTHORS="${AUTHORS//$'\n'/'%0A'}"
+          AUTHORS="${AUTHORS//$'\r'/'%0D'}"
+          echo "::set-output name=authors::$AUTHORS"
+          
       - name: Get release notes
         id: get-release-notes
         run: |
-          NOTES=$(git log ${{ steps.get-last-tag.outputs.last }}..HEAD --pretty="%an - %b (%s)[%h]" --merges | sed -e 's!Merge pull request !!')
+          NOTES=$(git log ${{ steps.get-last-tag.outputs.last }}..HEAD --pretty=" - %b (%s)[%h]" --merges | sed -e 's!Merge pull request !!')
           NOTES="${NOTES//'%'/'%25'}"
           NOTES="${NOTES//$'\n'/'%0A'}"
           NOTES="${NOTES//$'\r'/'%0D'}"
           echo "::set-output name=notes::$NOTES"
           
-      
+      - name: Install Python dependencies
+        id: install-python
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyGithub
+          pip install invoke
+          
+      - name: Create Updates
```

Comment:
haha. i should have commented on this so we could get a thread. i think moving this under .github is 👍  ive done .github/scripts for the demotron repo. im pretty open as to the exact path, but as long as its under .github, sounds good to me.

Reply:
ha i don't know why you can't just make a thread without commenting 🙈  excellent ye that sounds much better, keep it all neat and tidy, thanks